frontend/src/components/Secrets/GSecretDialog.vue

```diff
@@ -358,17 +326,10 @@ export default {
             name: this.name,
             namespace: this.namespace,
           },
+          provider: {
+            type: this.vendor,
```

Comment:
isn't vendor not a another alias (like infrastructureKind etc) for providerType? If yes, we should also use providerType accordingly

Reply:
Yes, I will replace vendor with providerType in all secret dialogs